spec/requests/portfolio_items_spec.rb

```diff
@@ -12,6 +13,20 @@
     allow(ServiceCatalog::ServicePlans).to receive(:new).with(portfolio_item.id.to_s).and_return(svc_object)
   end
 
+  describe 'DELETE admin tagged /portfolio_items/:portfolio_item_id' do
+    let(:valid_attributes) { { :name => 'PatchPortfolio', :description => 'description for patched portfolio' } }
+
+    context 'when :portfolio_item_id is valid' do
+      before do
+        delete "/api/v0.0/portfolio_items/#{portfolio_item_id}", :headers => admin_headers, :params => valid_attributes
+      end
+
+      it 'deletes the record' do
```

Comment:
@syncrou Do we need a TODO to add a test as a non admin user it should fail. Since we don't have any constraints check now we would have to wait. Also a GitHub issue might help

Reply:
Added a TODO that points to this Issue: https://github.com/ManageIQ/service_portal-api/issues/85